src/extension.ts

```diff
@@ -277,11 +288,11 @@ export class Extension {
                 }]
             },
             OnDeviceComponent: {
-                // uncomment for debugging
-                // logLevel: 'verbose',
-                // serverDebugLogging: true,
+                logLevel: enableDebugging ? 'verbose' : undefined,
+                serverDebugLogging: enableDebugging,
                 disableTelnet: true,
-                disableCallOriginationLine: true
+                disableCallOriginationLine: true,
+                callbackListenPort: config.rdbCallbackPort
```

Comment:
Just checking, is it ok that this might be undefined sometimes? I don't see any default being set in the config provider. I'm assuming rta will fall back to a random port in that situation?

Reply:
correct. If it's undefined in the config then it will use the auto find port logic